src/components/page-sidebar/sidebar-tab.scss

```diff
@@ -0,0 +1,51 @@
+@import "../vars.scss";
+
+.sideBarTab {
+  cursor: pointer;
+  width: 75px;
+  min-height: 70px;
+  text-align: center;
+  padding: 8px;
+  border-radius: 3px 0 0 3px;
+  background-color: $cc-orange-light1;
+  fill: white;
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  left: -91px;
+  position: absolute;
+
+  &:hover {
+    box-shadow: -2px 2px 10px $cc-charcoal-light1;
+    background-color: $cc-orange-light2;
+  }
+  &:active {
+    box-shadow: -2px 2px 10px $cc-charcoal-light1;
+    background-color: $cc-orange-light3;
+  }
+
+  .icon {
+    height: 30px;
+    width: 30px;
+    background-color: $cc-orange-light4;
+    fill: $cc-orange-light1;
+    border: 1px solid $cc-orange-light1;
+    border-radius: 50%;
+    box-sizing: border-box;
+    margin-left: 20px;
+    transition-property: transform;
+    transition-duration: .25s;
```

Comment:
Is this the close button?  Not sure if it goes here, but I would add a cursor: pointer to the close button to indicate clickability.

Reply:
This is for the arrow in the tab. The close button is in the sidebar-panel. I added cursor: pointer to both. And removed the border to the arrow. Kept the border radius to hide the div background to match the tab.